server/pulp/server/db/connection.py

```diff
@@ -91,12 +100,25 @@ def initialize(name=None, seeds=None, max_pool_size=None, replica_set=None, max_
                                                                 itertools.repeat(32))
 
         if seeds != '':
-            seeds_list = seeds.split(',')
             if len(seeds_list) > 1 and not replica_set:
                 raise PulpCodedException(error_code=error_codes.PLP0041)
             while True:
                 _CONNECTION = _connect_to_one_of_seeds(connection_kwargs, seeds_list, name)
                 if _CONNECTION:
+                    db_version = semantic_version.Version(_CONNECTION.server_info()['version'])
+                    if cmp(db_version, MONGO_MINIMUM_VERSION) == -1:
+                        raise RuntimeError(_("Pulp requires Mongo version %s, but DB is reporting"
```

Comment:
I'd be clear here that we require that version *or greater*. We don't want someone to be confused that we might require one specific version.